custom_components/micro_air_easytouch/climate.py

```
@@ -131,29 +136,38 @@ def current_fan_icon(self) -> str:
+                self._state = self._data.decrypt(json_payload.decode('utf-8'))
+                _LOGGER.debug("Initial state fetched: %s", self._state)
+                self.async_write_ha_state()
             else:
-                self._state = {}
-                _LOGGER.warning("Failed to send command for initial state")
+                # Preserve last known state instead of clearing it
+                if not self._state:
+                    _LOGGER.warning("No payload received for initial state and no previous state available")
+                else:
+                    _LOGGER.warning("No payload received for initial state, preserving last known state")
         except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)